api/src/pcapi/routes/native/v1/serialization/bookings.py

```diff
@@ -87,6 +87,11 @@ class Config:
         orm_mode = True
 
 
+class ExternalBookingResponse(BaseModel):
+    barcode: str
+    seat: Optional[str]
+
+
```

Comment:
```suggestion

    class Config:
        orm_mode = True
```
Pour la sérialisation